kernel32.go

```diff
@@ -299,3 +301,22 @@ func SystemTimeToFileTime(lpSystemTime *SYSTEMTIME, lpFileTime *FILETIME) bool {
 
 	return ret != 0
 }
+
+// This will get added to x/sys/windows at some point, but we need it now
+// See https://github.com/golang/go/issues/11147
+func FormatMessage(flags uint32, msgsrc uintptr, msgid uint32, langid uint32, buf []uint16, args *byte) (n uint32, err error) {
+	var _p0 *uint16
+	if len(buf) > 0 {
+		_p0 = &buf[0]
+	}
+	r0, _, e1 := syscall.Syscall9(formatMessage, 7, uintptr(flags), uintptr(msgsrc), uintptr(msgid), uintptr(langid), uintptr(unsafe.Pointer(_p0)), uintptr(len(buf)), uintptr(unsafe.Pointer(args)), 0, 0)
```

Comment:
Unfortunately this is the convention - you pass `Syscall9` a pointer to the method you're calling, the number of non-null arguments, and then you pad out the list of arguments to 9 using nulls. 

![](https://s-media-cache-ak0.pinimg.com/originals/e1/59/60/e15960429e2ef92fcdbc2faa7352d5fa.gif)